source/helpers/conventional-commits.test.ts

```diff
@@ -0,0 +1,21 @@
+import {test, assert} from 'vitest';
+
+import {parseConventionalCommit} from './conventional-commits.js';
+
+function assertRegexDeepStrictEqual(input: string, expected: [string, string | undefined]) {
+	const result = parseConventionalCommit(input);
+	assert.deepStrictEqual(result, [input, ...expected]);
+}
+
+test('parseConventionalCommit', () => {
+	assertRegexDeepStrictEqual('fix: Commit message', ['fix', undefined]);
```

Comment:
```suggestion
	assertRegexDeepStrictEqual('fix: Commit message', ['fix: ', 'fix', undefined]);
```

this will tix the assertion. 

To answer a previous concern you had: it’s true that this will match `a: ` but then we have the whitelist of types, which `a` will fail. I suppose we can exclude commits that are titled exactly `fix: ` but I reckon that’s an extremely rare occurrence. 

At some point later I’ll move the `types` map to the helpers and have this parser return exactly `{match: 'chore(release): ', label: 'Chore: release'}` but that’s not necessary for now. It’s best to wrap up this PR 😃

Reply:
The argument is returned, so it has to be included in the tests and not ignored. The point of tests is to validate the entire result, not part of it